packages/stark-core/src/util/validation-class.util.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * Class used to validate that other classes are not undefined and/or not
```

Comment:
Perhaps this can be like;

"Class containing utils to be used for class validation decorators of the 'class-validator' library"